src/lib.rs

```diff
@@ -467,28 +477,34 @@ impl Octocrab {
 
     /// Send a `PUT` request to `route` with optional query parameters,
     /// returning the body of the response.
-    pub async fn put<R, A, P>(&self, route: A, parameters: Option<&P>) -> Result<R>
+    pub async fn put<R, A, P, B>(&self, route: A, parameters: Option<&P>, body: Option<&B>) -> Result<R>
```

Comment:
Instead of adding an argument I would replace `parameters`, it's not being used anywhere and then it matches `post`
```suggestion
    pub async fn put<R, A, B>(&self, route: A, body: Option<&B>) -> Result<R>
```

Reply:
Are you sure? It seems like it is used when locking an issue ([GitHub docs](https://developer.github.com/v3/issues/#lock-an-issue), [code in Octocrab](https://github.com/XAMPPRocky/octocrab/blob/f011c2d23ce9e54e76a42eef83c625a993e640e0/src/api/issues.rs#L160-L168))

Reply:
Yeah, my code is actually wrong, and this would fix it. Locking an issue wants you to send a JSON body not query parameters.